app/utils/assignments.js

```diff
@@ -1,6 +1,6 @@
 const BASE_URL = 'https://organize.berniesanders.com';
 
 export function getAssignments () {
-  return fetch(`${BASE_URL}/contact-assignments.json`)
+  return fetch(`${BASE_URL}/contact-assignments.json?platform=android`)
```

Comment:
Any changes in the response data that i need to address else where in the app?

Or is this good to drop right in?